src/App.js

```diff
@@ -17,9 +18,10 @@ class App extends React.Component {
         </button>
         <Questions/>
         <Reviews/>
+        <Related />
       </>
     );
   }
 }
 
-export default hot(App);
```

Comment:
This could be useful during development.
Looked this up on stackoverflow, seems pretty neat.
"React Hot Loader is a React library used to edit your code and automatically reflect that change in your running app while keeping the components mounted, which means you don't need to refresh your browser and you won't lose your current state. It's pretty convenient as you won't have to go step-by-step to test a specific state in your app every time you make a change in the code."


https://stackoverflow.com/questions/42077671/what-is-react-hot-loader